### ISSUE TITLE
Accessibility problems

### ISSUE THREAD

**Issue by lschmelzeisen** [semanux/website#8]
Some parts of our website are not as accessible as they could be:

- [ ] Our primary `$color-accent-1` (used mainly for clickable stuff like links) `#3B7C97` has an insufficient contrast ratio with our background color(s). This is both true for the `$color-main-background` (background of main content, i.e., `#EEF6F6`), but also for `$color-nearly-white` (background of header/footer, i.e., `#FAFAFA`). For reference, we currently fail both the `WCAG AA` and `WCAG AAA` tests for `normal text`, see [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/).

**Comment by lschmelzeisen** [comment]
On a small test, the site seems decently navigable with just a keyboard, but should also test this for other input devices, and especially screen readers.

**Comment by lschmelzeisen** [comment]
A complete redesign of the website is on the way, which should resolve these. Therefore I'm closing this for now. Feel free to open, if you think this is urgent or if the problem is still a thing after the redesign.

For reference, https://wave.webaim.org/ seems to be an excellent tool to test web site accessibility.
